synapse/models/files.py

```diff
@@ -200,7 +203,41 @@ def getModelDefs(self):
                 ('file:filepath', ('comp', {'fields': (('file', 'file:bytes'), ('path', 'file:path'))}), {
                     'doc': 'The fused knowledge of the association of a file:bytes node and a file:path.',
                 }),
-
+                ('file:mime:pe:section', ('comp', {'fields': (
+                        ('file', 'file:bytes'),
+                        ('name', 'str'),
+                        ('sha256', 'hash:sha256'),
+                    )}), {
+                    'doc': 'The fused knowledge a file:bytes node containing a pe section.',
+                }),
+# TODO get type/langid figured out
+#                ('file:mime:pe:resource', ('comp', {'fields': (
+#                        ('file', 'file:bytes'),
+#                        ('type', 'typecode'),
+#                        ('langid', 'langcode'),
+#                        ('resource', 'file:bytes'))}), {
+#                    'doc': 'The fused knowledge of a file:bytes node containing a pe resource.',
+#                }),
+                ('file:mime:pe:export', ('comp', {'fields': (
+                        ('file', 'file:bytes'),
+                        ('name', 'str'))}), {
```

Comment:
@whippit @therealsilence  i keep thinking about this and it feels like this is a `it:dev:str` since it is a developer chosen string and makes the exported name linkable.